statsd/requirements-dev.txt

```diff
@@ -0,0 +1 @@
+datadog-checks-dev
```

Comment:
do we need this? You added it in your tox config so it should be there already.

Reply:
I don't think I have this added in the tox config

Reply:
https://github.com/DataDog/integrations-core/pull/1877/files#diff-1c2462382f6cd726616c48b04684c65bR11

Reply:
They are different packages.

Reply:
oh boy... my eyes are failing me again 😞